addons/binding/org.openhab.binding.tellstick/README.md

```diff
@@ -1,104 +1,151 @@
- 
-## Supported Things
-
-This binding implements two different API:  
-**1)** *Telldus Core* which is a local only interface supported by USB based device. <br>
-**2)** *Telldus Live* which is a REST based cloud service maintained by Telldus. <br>
-3) (According to [Telldus](http://developer.telldus.com/blog/2016/01/21/local-api-for-tellstick-znet-lite-beta) are they working with a local REST based API for the new Z-Wave devices. This is currently **NOT** supported by this binding.)
+All of these 433 MHz devices is one-way, so some versions of the Tellstick monitoring the air to keep the state of all devices.
 
-Depending on your Tellstick model different API methods is available: 
+The latest versions have also implemented Z-Wave as transmission protocol which open up for more robust transmission due two-ways communication.
 
-<table>
-<tr><td><b>Model</b></td> <td><b>Telldus Core</b></td> <td><b>Telldus Live</b></td> <td><b>Verified working with openHAB</b></td></tr>
-<tr><td>Tellstick Basic</td><td>X</td><td>X</td></tr>
-<tr><td>Tellstick Duo</td><td>X</td><td>X</td><td>X</td></tr>
-<tr><td>Tellstick Net</td><td></td><td>X</td></tr>
-<tr><td>Tellstick ZNet Lite</td><td></td><td>X</td></tr>
-<tr><td>Tellstick ZNet Pro</td><td></td><td>X</td><td>X</td></tr>
-</table>
+![Tellstick](doc/tellstick_duo.jpg)
 
+## Supported Things
 
 This binding supports the following thing types:
 
-* Telldus Core Bridge
-* Telldus Live Bridge
-* Dimmable Device
-* Switchable Device
-* Sensors
-
-## Binding Configuration
+* *Dimmable Device* - Usually for controlling lamps.  `dimmer`
+* *Switchable Device* - On/Off only could be lamps or other electronic equipment. `switch`
+* *Sensors* - Often temperature-, barometic- and humidity-sensors. `sensor`
 
----
+Additionally the binding have two types of bridge things which correspond to available API types:
+* *Telldus Core Bridge* - Oldest API, used by USB devices. `telldus-core`
+* *Telldus Live Bridge* - Telldus Cloud service, all devices with online access. `telldus-live`
 
-#####For Telldus Core only#####
-
-*First of all you need to make sure that your JVM is matching your installed Telldus Center. This normally means openHab must run on a 32bit JVM for windows and a 64bit JVM for linux. For windows the binding is hardcoded to look for Telldus Center in Programs Files ("C:/Program Files/Telldus/;C:/Program Files (x86)/Telldus/"). If you have trouble getting the telldus core library to work you can modify the library path using*
+***Switchbased sensors workaround*** <br>
+*Some 433MHz magnetic & PIR sensors for example magnetic door sensors are detected as a regular `switch` things instead of a separate type. There is technically no way of distinguish them apart from regulur `switch` things. For using them as sensors only (not paired to a lamp) please consult the workaround in the channel section.*
 
 ## Discovery
+Devices which is added to *Telldus Core* and *Telldus Live* can be discovered by openHAB.
```

Comment:
please add an empty line after every headline as Jekyll otherwise breaks the format